wp-content/themes/goonj-crm/assets/styles/lg.scss

```diff
@@ -37,16 +37,16 @@
     }
     
     .crm-container {
+        width: 720px !important;
         padding: 24px;
         
         af-form {
-            width: 520px !important;
             margin-left: auto;
             margin-right: auto;
         }
 
         .select2-search-choice-close {
-            padding-top: 8px !important;
+            padding-top: auto !important;
```

Comment:
The value `auto` is not a valid value for the `padding-top` property. See: https://developer.mozilla.org/en-US/docs/Web/CSS/padding-top#syntax

Reply:
Corrected this